.travis.yml

```diff
@@ -8,7 +9,14 @@ env:
   - CF_SPACE="production"
   - CF_USERNAME="govuk-design-system-deploy-production@digital.cabinet-office.gov.uk"
   # CF_PASSWORD
-  - secure: kyCYOisfZhZUYgg7XqOydUObKgxxwZkc5JCA+lzgrCY8d71hyX8uSh2jEnSE4A7Tq+4BR6oFg2C9kKSyq6G3wkKkRQKroWvQD5ylNWn2vdhyCRcUKqOu4VStNq6th/1FVZHlSf5MumLtULVsWgvkDt7pwEHAJawKISMpVnM5RGO3VHlF40UfVfM9GQN5T/yEXm0FbdZ8k0wFQ0VHE0vskW8BgWi/U13GyqQB9pl/fr3I6Fbhk0u++UksSrEZhwfz/oXezcJ4wuSwYtBnLFHFIJiZas5Cwy1uJTvsP6bzYl4llDvhWwCojJtlumBIq0EYti66CQRytUQU0wbNyzhdFe33cWwodPOs2nrb44QYWhEbXLDqaZ2tVDaIMFugKWuWru+jwbIQfng6irIS7vbewpW83l7tg5BfSS4Br4KKS+/xNQpbJ6lm5PUuE7pgnGPFLI8/SEcwkhjNv9A0NEr3pmVI+LNUkdDcrz1t62x35GNkJ4B0OoOXjNmEEshI7oWKvGKFhQZoYdgTRYp9aFkeShgu+JLbhSoaJLDsxkHAmfyRLbxyy4miLDx7XqiAcWOz9qSZYxG5BrAmwf4xMihNuVQFjGe5VY/GYQXBgXPfP1GeD6JaaYRLf7WXbL53hzWS/xQqinbeL+EcXX9rLCTUnVLr5xzK3IvEygy+uoHIIKE=
+  - secure: UYvRRq3/WSAF5udIroJRoN0i4ha04K9tWf5ozXAidP9oF1o3kAdz7dMlb/UWE10KCki8WwYSKufsJknSC7yygg6BgkajpUFWZc36XWUgyIlXEti3kxjq8GhHUtVQpdH/Mp702DgzJCgA4Bwj9TlfkyP+PIYW4PfljRllJtPzqgxSur++23Q+kMmvA8T/GOEyab72ZjEMQmonl0Vxf6UWx/y7/4+XLj34OTzoYQ18utWfH9o9i1KUA8dYFCT3oCauGXF8Ra6iOPmNhBjvKrT9+foyYvfbwtL+o+tcbyBQM+p2toWUC2E5e+gIyed+woNnUMFGjGzkwzRe5evhH7RbssGMnkAHn19EGht+Ycdo5Wqh06kmb6sGVFa5EfvuX8AUHaOJrjBjS4ojHA9rmrkCrLvXpKECJi/NK2if5mk39ULtmtUzBBAjgPY3ZY34wweKrRgKz1Q2+Z59nJuX4/UM/KP3lrG0IWKIqdcrnoRaXsWADmf512zFw699+3rre6Bv7h08cyJZWAPWFhZiFibFOQXpT7uVeGyF2VcNrDjVzOiUFOWb7+5IRLi3Volp9o5OHuH1aOw3vNb2KqbIvJAFcQJF4l7c8Jopp6obeyBtXsupAVOTKZQnc9rokwDvyLbCNfseXXWNPekRD9+Ey8asSM+NauWLASFIWxNrRj6M5m0=
+
+before_install:
+  - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
+
+install:
+  - bundle install
```

Comment:
How did this work before? Does bundle install get run automatically in a ruby environment?

Reply:
before making this change travis failed on build exec middleman with 127

Reply:
Looks like Travis [runs bundle install automatically](https://docs.travis-ci.com/user/languages/ruby/#Bundler) if you have a Gemfile and don't have an install command specified.

Might be worth using the retry flag that they specify there to make the process slightly more resilient to network interruptions.

Reply:
hmm, we tried just having cache:bundler and it failed (hoping it would be intelligent enough to know that it needs to do bundle install.
I guess we need to tell it where the gemfile is

Reply:
it flat out fails without bundle install, even with travis_retry

Reply:
Sorry, I wasn't very clear – I was agreeing that you need to add this line (because you're now specifying an `install` command and that overrides the default behaviour) but was suggesting we use the same arguments (`bundle install --jobs=3 --retry=3`) to defend against network interruptions etc.

Reply:
oh sorry, missunderstood. yeah will add the flags